assets/js/base/hooks/checkout/use-checkout-events.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import useCheckoutContext from '@woocommerce/base-context/checkout-context';
+
+export const useCheckoutEvents = () => {
```

Comment:
Why do we need to wrap out context by another layer of abstraction? Can’t we directly use the context? we would have one less hook to maintain and document

Reply:
It allows for controlling what slices of the context are potentially exposed publicly.  I think we want to guard what can control the cart.  Some slices of the context will be strictly for internal use, while others might be exposed publicly.

Another reason is this allows for potential flexibility in how these individual slices are derived in the future (they may may not come from the entire checkout context).  This would potentially reduce the refactoring needed because implementations are exposed to a smaller interface.